test/shared_setup.py

```diff
@@ -313,7 +313,10 @@ def read_setup_cfg():
         os.chdir(self.tmpdir)
 
         # test with minimal target
-        vsc_setup.build_setup_cfg_for_bdist_rpm({})
+        target = {
+            'description_file': 'README.md',
```

Comment:
why do you need to set this. it's the default value right?

Reply:
This is only needed in this unit test with an artificial target. In a real scenario the `description_file` attribute will be set by the call to `locate_readme()` in `action_target()`, which looks for the actual README file in the repo directory. The only case where a target would have no `description_file` is if there is no README, which will error out before reaching `build_setup_cfg_for_bdist_rpm()`.